app/lib/translation_service.rb

```diff
@@ -17,6 +17,12 @@ def self.configured
     end
   end
 
+  def self.configured?
+    configured.present?
```

Comment:
Probably don't need to instantiate a new class just to check if `DEEPL_API_KEY` or `LIBRE_TRANSLATE_ENDPOINT` are set.